### DISCUSSION TITLE
How effective is SumatraPDF?

### DISCUSSION THREAD

**Discussion by AndriyIl** [sumatrapdfreader/sumatrapdf#4648] [General]
I've been using Sumatra for many years and really like it for its optimization and speed, especially when compared to an adobe reader.

But in the latest versions, I noticed that it became slower. I mostly open pdf files with drawings or visualizations, so they are usually heavy files. Recently, I opened such a file, not very big, 2 MB, one page, but with small elements. And it took a relatively long time to open, and then when zooming in and out, it also froze. I decided to look in the Task Manager to see the CPU load during opening and zooming. The total load was up to 10%, I looked at the load on each core and found that only one core was used. 

Do you plan to add deeper support for processor cores?

**Comment by kjk** [comment]
Generally speaking, SumatraPDF should be becoming faster on each release.

If you have a specific file that opens slowly, please open an issue
